help/scim.md

```diff
@@ -70,6 +70,20 @@ Zulip's SCIM integration has the following limitations:
      * **givenName**
      * **familyName**
 
+1. **Optional:** If you'd like to also sync [user role](/help/roles-and-permissions),
+   you can do it by by adding a custom attribute in Okta. Go to the **Profile Editor**,
+   click into the entry of the SCIM app you've just set up and **Add Attribute**.
+   Configure the following:
+    * **Data type**: `string`
+    * **Variable name**: `role`
+    * **External name**: `role`
+    * **External namespace**: `urn:ietf:params:scim:schemas:core:2.0:User`
```

Comment:
This is for this popup in Okta:
![image](https://github.com/zulip/zulip/assets/45007152/ee875e53-1caf-478a-b765-3aabebd8350b)